diff --git a/release/src/router/lighttpd-1.4.39/Makefile.in b/release/src/router/lighttpd-1.4.39/Makefile.in
index 9ac34073fa..a03397f731 100755
--- a/release/src/router/lighttpd-1.4.39/Makefile.in
+++ b/release/src/router/lighttpd-1.4.39/Makefile.in
@@ -656,12 +656,12 @@ uninstall: uninstall-recursive
 #20160128 Sungmin add
 copy-prebuild:
 	@if test -f "src/mod_smbdav.c"; then :; else \
-		cp prebuild/mod_smbdav.so.l src/.libs/mod_smbdav.so; \
-		cp prebuild/mod_aidisk_access.so.l src/.libs/mod_aidisk_access.so; \
-		cp prebuild/mod_aicloud_sharelink.so.l src/.libs/mod_aicloud_sharelink.so; \
-		cp prebuild/mod_aicloud_auth.so.l src/.libs/mod_aicloud_auth.so; \
-		cp prebuild/mod_aicloud_invite.so.l src/.libs/mod_aicloud_invite.so; \
-		cp prebuild/mod_query_field_json.so.l src/.libs/mod_query_field_json.so; \
+		cp prebuild/GT-AX6000/mod_smbdav.so src/.libs/mod_smbdav.so; \
+		cp prebuild/GT-AX6000/mod_aidisk_access.so src/.libs/mod_aidisk_access.so; \
+		cp prebuild/GT-AX6000/mod_aicloud_sharelink.so src/.libs/mod_aicloud_sharelink.so; \
+		cp prebuild/GT-AX6000/mod_aicloud_auth.so src/.libs/mod_aicloud_auth.so; \
+		cp prebuild/GT-AX6000/mod_aicloud_invite.so src/.libs/mod_aicloud_invite.so; \
+		cp prebuild/GT-AX6000/mod_query_field_json.so src/.libs/mod_query_field_json.so; \
 	fi
 
 install-am: all-am
